runtime/src/lib.rs

```diff
@@ -491,12 +565,12 @@ impl pallet_utility::Config for Runtime {
 }
 
 parameter_types! {
-    pub const MinDomainLength: u32 = 7;
+    pub const MinDomainLength: u32 = 5;
     pub const MaxDomainLength: u32 = 63;
 
     pub const MaxDomainsPerAccount: u32 = 100;
     // TODO This value should be removed later, once it will be possible to purchase domains.
-	pub const MaxPromoDomainsPerAccount: u32 = 3;
+	pub const MaxPromoDomainsPerAccount: u32 = 10;
```

Comment:
Not sure that is what you wanted, because now people, who have SUB, can obtain 7 more domains.